services/ui-src/src/verbiage/forms/adminBannerForm.ts

```diff
@@ -4,6 +4,40 @@ export const form = {
     mode: "onChange",
   },
   fields: [
+    {
+      id: "abf-radio",
+      type: "radioList",
+      props: {
+        name: "Choice List Radio Name",
+        type: "radio",
+        label: "Choice List Radio Label",
+        choices: [
+          { label: "Choice 1", value: "A" },
+          { label: "Choice 2", value: "B" },
+          { label: "Disabled choice 3", value: "C" },
+        ],
+      },
+      validation: {
+        type: "array",
+      },
+    },
+    {
+      id: "abf-check",
+      type: "checkboxList",
+      props: {
+        name: "Choice List Checkbox Name",
+        type: "checkbox",
+        label: "Choice List Checkbox Label",
+        choices: [
+          { label: "Choice 1", value: "A" },
+          { label: "Choice 2", value: "B" },
+          { label: "Disabled choice 3", value: "C" },
+        ],
+      },
+      validation: {
+        type: "array",
+      },
+    },
```

Comment:
Remember to take these out before you finish up. 

Also will you check out validation for these? Adding in `required` might not work directly, but we could try using `minLength` of 1 or something similar.

Reply:
```
validation: {
  type: "array",
  options: { min: 1 },
  errorMessages: {
    min: "Field is required",
  },
},
```
This will check to make sure that the choicelist is filled out with at least 1 option

Reply:
minLength does not work here. Despite the docs telling me it would 🙃

Reply:
So this validation does not technically work because the field is not being registered with the form correctly, which is kind of a larger issue we need to look into. That's also why the date field validation is not working correctly. We also don't want fields to be showing error messages before the user interacts with them. For right now we can merge this PR and move forward but we need to come back to this next.